### PR TITLE
Add discussion of matplotlib magics to jupyter notebook introduction.

### PR DESCRIPTION
**Pull request by JuanCab**
I added discussion of %matplotlib notebook and %matplotlib inline) to the introduction to jupyter notebook.  This addresses and hopefully resolves issue #26

### CODE REVIEW

**Comment by kelle** [comment]
@shannnnnyyy , since you are leading this session, could you pls review this PR. thanks!

**Comment by shannnnnyyy** [comment]
yes - will review this this evening.

File: 02-PythonIntro/00. Using the Jupyter notebook.ipynb [hunk 1]
```diff
@@ -111,27 +110,848 @@
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The plotting package we will use during today' workshop is called `matplotlib`.  To use it within Jupyter notebooks we have to first apply a `matplotlib magic` command to tell `matplotlib` where to direct the plotting.  When using Jupyter notebooks, the two most common choices are\n",
+    "\n",
+    "1. `%matplotlib notebook`: generates an *INTERACTIVE* plot (e.g. - allowing you to pan/zoom into portions of the plot using the controls below the plot and download the resulting plot)\n",
+    "2. `%matplotlib incline`: generates an non-interactive plot \"inline\" underneath the cell once the plot is generated.\n"
```
Comment (shannnnnyyy) [inline 02-PythonIntro/00. Using the Jupyter notebook.ipynb]: typo - 'incline' should be 'inline'

**Comment by JuanCab** [comment]
I fixed the typo.  I did leave out the "example" because below I state they can try to change the matplotlib magics statement and re-execute.  I figured it was more instructional for them to see that they can change commands but the effects don't take place until they re-execute the commands.  If you feel it would be better to have both versions in there, I can do that as well, but it was a toss up for me.

File: 02-PythonIntro/00. Using the Jupyter notebook.ipynb (hunk 1 above)
Comment (JuanCab) [reply 02-PythonIntro/00. Using the Jupyter notebook.ipynb]: Fixed.

Comment (JuanCab) [reply 02-PythonIntro/00. Using the Jupyter notebook.ipynb]: Fixed.

**Comment by kelle** [comment]
@shannnnnyyy Looks like Juan addressed your issues. Is this ready to merge?

**Comment by kelle** [comment]
Going ahead and merging.

**Pull request merged**